functions/packages.bash

```diff
@@ -396,27 +396,50 @@ find_setup() {
 ##    knxd_setup()
 ##
 knxd_setup() {
-  local introText="This will install and setup kndx (successor to eibd) as your EIB/KNX IP gateway and router to support your KNX bus system.\\n\\nThis routine was provided by 'Michels Tech Blog': https://bit.ly/3dzeoKh"
-  local successText="Setup was successful.\\n\\nPlease edit '/etc/default/knxd' to meet your interface requirements. For further information on knxd options, please type 'knxd --help'\\n\\nFurther details can be found under: https://bit.ly/3dzeoKh\\n\\nIntegration into openHAB 2 is described here: https://github.com/openhab/openhab/wiki/KNX-Binding"
+  local introText="This will install and setup kndx as your EIB/KNX IP gateway and router to support your KNX bus system.\\n\\nNOTE: You don't need this if you connect via an IP interfacte or router to your KNX installation. This package is to turn an USB or serial interface into an IP interface.\n\nNOTE: openHABian changed from building and installing latest source to installing the knxd package provided by several distrubutions."
```

Comment:
will it also _setup_ knxd? If no remove from this text

2 typos:
interfacte 
distrubutions

Reply:
Ok, thanks. Wish I could find a spell checker for strings bash scripts....
Will merge once checks have completed.